notebooks/06_Serving/06_02_Kfserving.ipynb

```diff
@@ -0,0 +1,493 @@
+    "                           tensorflow=V1alpha2TensorflowSpec(\n",
+    "                             storage_uri='gs://kfserving-samples/models/tensorflow/flowers-2',\n",
+    "                             resources=V1ResourceRequirements(\n",
+    "                                 requests={'cpu':'100m','memory':'1Gi'},\n",
+    "                                 limits={'cpu':'100m', 'memory':'1Gi'}\n",
+    "                             )\n",
+    "                           )\n",
+    "                         )\n",
+    "                       )\n",
+    "\n",
+    "KFServing.rollout_canary('flower-sample', canary=canary_endpoint_spec, percent=10,\n",
+    "                         namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Rollout more traffic to canary of the InferenceService\n",
+    "Rollout traffice percent to 50% to canary version."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing.rollout_canary('flower-sample', percent=50, namespace=namespace,\n",
+    "                         watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Users can query service afterwards. For example, users query 10 times and they can see 5 times from raw endpoint and 5 times from canary endpoint."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Promote Canary to Default"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing.promote('flower-sample', namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Delete the InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing.delete('flower-sample', namespace=namespace)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for Kfserving YAML\n",
+    "## Setup\n",
+    "1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).\n",
+    "2. Your cluster's Istio Ingress gateway must be network accessible.\n",
+    "\n",
+    "Note: You should execute all the code blocks in your local environment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Label kubeflow namespace so you can run inference tasks in it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl label namespace kubeflow serving.kubeflow.org/inferenceservice=enabled"
```

Comment:
Why don't you have this for SDK section? Do you think it's better to move to prerequsite section? Can you confirm the default value? I think by default, it should be enabled.

Reply:
Right, I put them together in the Notebook Prerequisites section.